BackendWebApp/ci/update.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#
+# Kill current deployed instance and start deployment script.
+#
+# Parameters:
+#   1: deploy configuration base path path.
+#   2: application port
+#   3: deploy jar artifact name, i.e. biostudy-20190430.jar
+#
+# Example execution
+#
+#  ./update.sh /ebi/teams/biostudies/backend/apps/webapp 8586 biostudy-20190430.jar
+#
+cd $(dirname $0)
+
+# kill current application in specified port
```

Comment:
I think the comment here should be something like:
Get the process id of the application running in the specified port